internal/cli/terraform.go

```diff
@@ -187,3 +217,51 @@ import {
 
 	return t.Execute(file, data)
 }
+
+func generateTerraformResourceConfig(ctx context.Context, outputDIR string) error {
+	absoluteOutputPath, err := filepath.Abs(outputDIR)
+	if err != nil {
+		return err
+	}
+
+	installer := &releases.ExactVersion{
```

Comment:
Should we get the latest version version that is >= 1.5.0 (within the 1.x major), instead of exactly the 1.5.0 version?

Reply:
In any case, would it make sense for this version to be configurable?

Reply:
We need at least terraform v1.5.0 to be able to generate the resource config files, but after that a user can use any terraform version => 1.0 to manage the configuration, so it wouldn't make much sense to make the version configurable IMO. 

Considering the fact that this new feature is still marked as "experimental" I wouldn't fetch the latest version either, as there might be differences in the behavior of the tool.